docs/intro/tutorial.rst

```diff
@@ -1,141 +1,446 @@
 .. _intro-tutorial:
 .. _page-objects:
 
-=====================
-web-poet on a surface
-=====================
+========
+Tutorial
+========
 
-In this tutorial, we’ll assume that web-poet is already installed on your
-system.
+In this tutorial you will learn to use web-poet as you write web data
+extraction code for book detail pages from `books.toscrape.com`_.
 
-We are going to scrape `books.toscrape.com <http://books.toscrape.com/>`_,
-a website that lists books from famous authors.
+.. _books.toscrape.com: http://books.toscrape.com/
 
-Creating a Page Object
-======================
+To follow this tutorial you must first be familiar with Python_ and have
+:ref:`installed web-poet <install>`.
 
-Let's create a Python file where we'll put our first Page Object implementation.
-This Page Object will be responsible for extracting book links from the book
-list page on `books.toscrape.com <http://books.toscrape.com/>`_.
+.. _Python: https://docs.python.org/
 
-.. code-block:: python
 
-    from web_poet.pages import WebPage
+Create a project directory
+==========================
+
+web-poet does not limit how you structure your web-poet web data extraction
+code, beyond the limitations of Python itself.
+
+However, in this tutorial you will use a specific project directory structure
+designed with web-poet best practices in mind. Consider using the same project
+directory structure in all your web-poet projects.
```

Comment:
I think the structure might depend on the framework used, e.g. it might be different in case of scrapy-poet. Should we say "similar" instead of "the same"?

Reply:
The structure looks fine overall, but I wonder what run.py method should contain in user projects.

Reply:
> I wonder what run.py method should contain in user projects.

Nothing, it is described as “a file specific to this tutorial” on its first mention.